packages/airnode-starter/scripts/make-request.ts

```diff
@@ -0,0 +1,30 @@
+import { getDeployedContract, getProvider, readIntegrationInfo } from '../src';
+
+async function fulfilled(requestId: string) {
+  const airnodeRrp = await getDeployedContract('@api3/protocol/contracts/rrp/AirnodeRrp.sol');
+  const provider = getProvider();
+  return new Promise((resolve) => provider.once(airnodeRrp.filters.FulfilledRequest(null, requestId), resolve));
+}
+
+// TODO: maybe just call the make-request and handle all logic there...
+async function main() {
+  const integrationInfo = readIntegrationInfo();
+  // TODO: What to do if this file is misssing?
+  const { makeRequest, printResponse } = await import(`../integrations/${integrationInfo.integration}/make-request.ts`);
+
+  console.log('Making request...');
```

Comment:
Nit: I think if some console output ends with "...", I think it should be followed up by another output that states it was successful. This isn't always obvious to the user

Reply:
Not sure I understand. There will be more output (`Waiting for fulfillment...`) and in the end there will be the request response.

Reply:
Ah ok. Ignore then